target/classes/META-INF/maven/car-app/car-app/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven Integration for Eclipse
```

Comment:
Do not commit the generated maven files or IDE related files, i.e. **target** folder.